src/com/cft/contactmerge/tests/MatchMakerTest.java

```diff
@@ -0,0 +1,24 @@
+package com.cft.contactmerge.tests;
+
+import java.util.*;
+import com.cft.contactmerge.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MatchMakerTest {
+
+    @Test
+    void MatchMaker() {
+        List<Contact> existingContacts = new ArrayList<Contact>();
```

Comment:
I'm not sure if this is going to be contacts from file 1, and contacts from file 2. If so... You may want to change it to fileOneContacts and fileTwoContacts. Just a suggestion but not a review breaker.

Reply:
The rules, as I have defined them so far, are different depending on the direction and so we need to know if something is the source of the merge or the target. Not a huge fan of "source" and "target". The names I've used in the class diagram is "ExistingContacts" which is the target of a merge and "ContactsToMerge" which is the source. Let me know if you can think of better names.